Examples/Objective-C/ViewController.m

```diff
@@ -126,7 +126,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         
         MGLPointAnnotation *destination = [[MGLPointAnnotation alloc] init];
         destination.coordinate = self.destination;
-        controller.destination = destination;
```

Comment:
Why was this line removed? Doesn’t this cause the preceding two lines to be useless?

Reply:
`destination` was deprecated eons ago.

Reply:
So shouldn’t the two lines above be removed too?